staging/src/k8s.io/apimachinery/pkg/util/net/interface_test.go

```diff
@@ -128,6 +128,9 @@ var (
 	routeV4    = []Route{ipv4Route}
 	routeV6    = []Route{ipv6Route}
 	bothRoutes = []Route{ipv4Route, ipv6Route}
+
+	preferV4 = []AddressFamily{familyIPv4, familyIPv6}
```

Comment:
move this to interface.go and use it instead of the hardcoded slices to improve readability.